conf/eslint-recommended.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
         "jsx-quotes": "off",
         "key-spacing": "off",
         "keyword-spacing": "off",
+        "layout-thrashing": "error",
```

Comment:
I don't think we should enable it in `eslint-recommended` -- changing `eslint-recommended` would be a breaking change. see https://github.com/eslint/eslint#semantic-versioning-policy

Reply:
@Aladdin-ADD Dint know what should be done in eslint-recommended.js file. now i did the required changes..